src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -189,16 +185,13 @@ namespace Google.Example.Library.V1
         /// </list>
         /// Default RPC expiration is 30000 milliseconds.
         /// </remarks>
-        public CallSettings GetShelfSettings { get; set; } = new CallSettings
-        {
-            Timing = CallTiming.FromRetry(new RetrySettings
-            {
-                RetryBackoff = GetDefaultRetryBackoff(),
-                TimeoutBackoff = GetDefaultTimeoutBackoff(),
-                RetryFilter = IdempotentRetryFilter,
-                TotalExpiration = Expiration.FromTimeout(TimeSpan.FromMilliseconds(30000)),
-            }),
-        };
+        public CallSettings GetShelfSettings { get; set; } = CallSettings.FromCallTiming(
```

Comment:
I've been wondering about a CallSettings.FromRetry, given how often we're likely to get this chaining.
We can add it later in a non-breaking way though.